articles/quickstart/backend/java-spring-security5/interactive.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Spring Boot API
+description: Secure your API using Spring Security 5 and Auth0
+interactive: true
+alias:
+  - spring security
+  - spring
+topics:
+  - quickstart
+  - backend
+  - spring
+files:
+  - files/application
+  - files/audience-validator
+  - files/security-config
+  - files/message
+  - files/api-controller
+github:
+  path: 01-Authorization-MVC
+---
+
+# Secure your API using Spring Security 5 and Auth0
+
+Auth0 allows you to quickly add authorization to your application. This guide demonstrates how to integrate Auth0 with any new or existing Spring Boot application.
```

Comment:
Is it intentional that we are adding back the reference to the SDK here? I believe this got removed because this Quickstart doesn't use that specific SDK. Can you verify @poovamraj  ?

Also curious why we want to change Authorization to Authentication and User Profile information again? This Quickstart is about adding Authorization to an API, not so much about Authentication or User Profile info.

Reply:
Updated.